src/test/java/io/camunda/testing/extensions/ZeebeAssertionsExtensionTest.java

```diff
@@ -0,0 +1,59 @@
+package io.camunda.testing.extensions;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import java.util.Optional;
+import org.camunda.community.eze.RecordStreamSource;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.mockito.Mockito;
+
+class ZeebeAssertionsExtensionTest {
+
+  @Nested
+  class MissingRecordStreamSource {
+
+    private ZeebeClient client;
```

Comment:
🔧 Maybe add a comment to this field, explaining why it is necessary for this test case, i.e. in order for `RecordStreamSource` to be the only one that is missing it needs to have all other required fields, incl `ZeebeClient`.